gpu.Dockerfile

```diff
@@ -13,8 +13,8 @@ COPY --from=nvidia /etc/apt/trusted.gpg /etc/apt/trusted.gpg.d/cuda.gpg
 
 # Ensure the cuda libraries are compatible with the custom Tensorflow wheels.
 # TODO(b/120050292): Use templating to keep in sync or COPY installed binaries from it.
-ENV CUDA_VERSION=9.2.148
-ENV CUDA_PKG_VERSION=9-2=$CUDA_VERSION-1
+ENV CUDA_VERSION=10.0.130
```

Comment:
Can you rebase your branch against master to remove the diff related to the cuda version change? 

It will make this PR easier to review. 

Thank you

Reply:
sure, will do. Thank you for letting me know.

Reply:
I have rebased the branch. Please let me know.